### ISSUE TITLE
propertyPath doesn't allow to access an array element

### ISSUE THREAD

**Issue by mogui** [fjogeleit/yaml-update-action#424]
Could you use something more robust than the custom implementaion, like JsonPath ?

https://github.com/fjogeleit/yaml-update-action/blob/036e48d849523a88238477939a74583b64de442a/src/action.ts#L96

**Comment by fjogeleit** [comment]
You can access array elements with .indexNumber like `containers.0.image`. Yes it is currently very basic. I will have a look on your suggestion.

**Comment by mogui** [comment]
Oh thanks it works! 
Yeah json path would be much more elagant, you can do smoething more complex  like `path.array[name=foo].bar`

**Comment by fjogeleit** [comment]
yeah, you're right, I will test it and see if it is backward compatible or how I can make it optional available. Don't want to break existing workflows.

**Comment by fjogeleit** [comment]
The new release has jsonpath support. Thanks for the suggestion:

https://github.com/fjogeleit/yaml-update-action/releases/tag/v0.7.0

**Comment by mogui** [comment]
Great!
